mailjet.php

```diff
@@ -866,13 +870,55 @@ public function activateAllEmailMailjet()
 	{
 		Configuration::updateValue('PS_MAIL_SERVER', $this->mj_mail_server);
 		Configuration::updateValue('PS_MAIL_SMTP_PORT', $this->mj_mail_port);
-		Configuration::updateValue('PS_MAIL_SMTP_ENCRYPTION', 'tls');
+		//Configuration::updateValue('PS_MAIL_SMTP_ENCRYPTION', 'tls');
 		Configuration::updateValue('PS_MAIL_USER', $this->account->API_KEY);
 		Configuration::updateValue('PS_MAIL_PASSWD', $this->account->SECRET_KEY);
 		Configuration::updateValue('PS_MAIL_METHOD', 2);
 		Configuration::updateValue('MJ_ALLEMAILS', 1);
+        
+        $account = (array) Tools::jsonDecode(Configuration::get('MAILJET'));
```

Comment:
Hi,

If you want to get an array instead of an object, you should add `true` for the second parameter.
Tools::jsonDecode calls json_decode, so you can have more information here: http://php.net/manual/fr/function.json-decode.php